README.md

```diff
@@ -4,7 +4,7 @@
 
 > This is a Leaderboard list app, setup using webpack and ES6 modules.
 
-![screenshot](./screen.JPG)
+![screenshot](./screen2.JPG)
 
 
```

Comment:
- [ ] Add clear instructions to follow to set up on a local computer. I have followed your instructions but I get an error when I run `npm run start`.
Remember to build your app with webpack, also indicate which files should be opened if any.